src/WebJobs.Script.Host/Program.cs

```diff
@@ -19,11 +23,38 @@ static void Main(string[] args)
 
             ScriptHostConfiguration config = new ScriptHostConfiguration()
             {
-                RootScriptPath = rootPath
+                RootScriptPath = rootPath,
+                FileLoggingEnabled = true
             };
 
+            // For local execution, allow reading secrets from a file. 
```

Comment:
This is optional.   EnvVars doesn't scale well as we get many secrets. For example, we may be binding to 10 different events hubs connections. 

---
In reply to: [54119905](https://github.com/Azure/azure-webjobs-sdk-script/pull/26#discussion_r54119905) [](ancestors = 54119905)